src/Main.java

```diff
@@ -1,5 +1,102 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        //Задание 1
+        int bananas = 3;
+        System.out.println("Бананов " + bananas + " кг");
+        byte  apples = 4;
+        System.out.println("Яблоков " + apples + " кг");
+        short tangerines = 5;
+        System.out.println("Мандаринов " + tangerines + " кг");
+        long oranges = 6;
+        System.out.println("Апельсинов " + oranges + " кг");
+        float lemous = 7;
+        System.out.println("Лимонов " + lemous + " кг");
+        double pears = 6;
+        System.out.println("Груш " + pears + " кг");
+        //Задание 2
+        float a = 27.12f;
+        System.out.println(a);
+        long b = 987_678_965_549L;
+        System.out.println(b);
+        float c = 2.786f;
+        System.out.println(c);
+        short d = 569;
+        System.out.println(d);
+        short e = -159;
+        System.out.println(e);
+        short f = 27897;
+        System.out.println(f);
+        byte g = 67;
+        System.out.println(g);
+        //Задание 3
+        byte LudmilaPavlovna = 23;
```

Comment:
Имена переменных в camelCase, начиная с буквы в нижнем регистре. Даже если это имя собственное
```suggestion
        byte ludmilaPavlovna = 23;
```